spec/support/helpers/flickr_helper.rb

```diff
@@ -0,0 +1,80 @@
+module FlickrHelper
+  def stub_flickr_requests
+    recent_response = { "photos": {
+      "page": 1,
+      "pages": 100,
+      "perpage": 10,
+      "total": "1000",
+      "photo": [
+          { "id": "22026223630",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.